src/info_command.c

```diff
@@ -74,6 +74,12 @@ static void renderIndexDefinitions(RedisModule_Reply *reply, IndexSpec *sp) {
     REPLY_KVSTR_SAFE("payload_field", rule->payload_field);
   }
 
+  if (rule->index_all) {
```

Comment:
Maybe we should also output here the memory usage specifically for the new inverted index in case it was enabled?

Reply:
I don't think we should add that here.
This may be added to the `FT.INFO` response, but I don't want it to create confusion with regarding the memory we use, since this memory is counted already in `sp->stats.invertedSize`